Makefile

```diff
@@ -1,11 +1,20 @@
+export CGO_ENABLED:=0
+
 VERSION=$(shell ./scripts/git-version)
+GOPATH_BIN:=$(shell echo ${GOPATH} | awk 'BEGIN { FS = ":" }; { print $1 }')/bin
 
 .PHONY: all
 all: build
 
 .PHONY: build
-build:
-	@go install -v github.com/coreos/terraform-provider-matchbox
+build: bin/terraform-provider-matchbox
+
+bin/%:
+	@go build -o bin/$* -v github.com/coreos/terraform-provider-matchbox
```

Comment:
right, but running `make bin/cat` will successfully build the same binary into `bin/cat`. normally parameterizing the target is for building many different binaries into different locations. you might as well hardcode:
```make
bin/terraform-provider-matchbox:
    @go build...
```
or better yet, put the `go build...` directly under `build`